Data.R

```diff
@@ -1,97 +1,131 @@
+#'---
+#' title: "[TRIPOD 1] Identify the study as developing and/or validating a multivariable prediction model, the target population, and the outcome to be predicted."
+#' author: 'Author One ^1,✉^, Author Two ^1^'
+#' abstract: |
+#'  | [TRIPOD 2] Provide a summary of objectives, study design, setting, participants, sample size, predictors, outcome, statistical analysis, results, and conclusions.
+#' documentclass: article
+#' description: 'Manuscript'
+#' clean: false
+#' self_contained: true
+#' number_sections: false
+#' keep_md: true
+#' fig_caption: true
+#' css: production.css
+#' output:
+#'   html_document:
+#'    code_folding: hide
+#'    toc: true
+#'    toc_float: true
+#' ---
+#'
```

Comment:
This header adds placeholders for title/authors/abstract and also improves formatting of the document. You can use this header, with minor modifications, as boilerplate in future scripts.